addons/point_of_sale/views/pos_assets_common.xml

```diff
@@ -107,10 +109,7 @@
 <template id="point_of_sale.assets_backend" name="hr assets" inherit_id="web.assets_backend">
     <xpath expr="." position="inside">
         <link rel="stylesheet" type="text/scss" href="/point_of_sale/static/src/scss/pos_dashboard.scss"/>
-        <link rel="stylesheet" href="/point_of_sale/static/src/css/pos_receipts.css"/>
```

Comment:
why do you remove this files?

Changes on the xml file here cannot be done in 14.0, as it won't impact existing DB